tests/testthat/test-module_teal.R

```diff
@@ -1,6 +1,6 @@
 # comment: srv_teal is exported so the tests here are extensive and cover srv_data as well.
 #          testing of srv_data is not needed.
-module_output_table <<- function(output, id) {
+module_summary_table <<- function(output, id) {
```

Comment:
you haven't changed this but out of curiosity: why double assignment here?

Reply:
`testthat::describe/testthat::it` evaluates in the own environment which is the sibling of the environment from `test-<filename>.R`. These testing functions are forcing you to test only public objects so even in tests it is required to somehow deliver helper function.

We choose to use `describe/it` instead of `test_that` to provide end-to-end testing and not to test private utilities (which are evaluated anyway within e2e test)